src/test/kotlin/com/petqua/application/product/review/ProductReviewFacadeServiceTest.kt

```diff
@@ -0,0 +1,229 @@
+package com.petqua.application.product.review
+
+import com.amazonaws.services.s3.AmazonS3
+import com.ninjasquad.springmockk.SpykBean
+import com.petqua.application.product.dto.ProductReviewCreateCommand
+import com.petqua.common.domain.findByIdOrThrow
+import com.petqua.domain.product.review.ProductReviewImageRepository
+import com.petqua.domain.product.review.ProductReviewRepository
+import com.petqua.exception.product.review.ProductReviewException
+import com.petqua.exception.product.review.ProductReviewExceptionType
+import com.petqua.test.DataCleaner
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldContain
+import io.mockk.verify
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.NONE
+import org.springframework.http.MediaType
+import org.springframework.mock.web.MockMultipartFile
+
+@SpringBootTest(webEnvironment = NONE)
+class ProductReviewFacadeServiceTest(
+    private val productReviewFacadeService: ProductReviewFacadeService,
+    private val productReviewRepository: ProductReviewRepository,
+    private val productReviewImageRepository: ProductReviewImageRepository,
+    private val dataCleaner: DataCleaner,
+    @SpykBean private val amazonS3: AmazonS3,
+) : BehaviorSpec({
```

Comment:
일관성 있게 정렬하면 좋을 것 같아요!
```suggestion

    @SpykBean
    private val amazonS3: AmazonS3,
) : BehaviorSpec({
```